### PR TITLE
* Implement playlist countdown per second

### PR DESCRIPTION
**Pull request by PikachuEXE**
**Pull Request Type**
Please select what type of pull request this is:
- [X] Feature Implementation

**Related issue**
https://github.com/FreeTubeApp/FreeTube/issues/1214

**Description**
Instead of just showing "Playing next video in N seconds. Click ..." 1 time
Show that message with remaining of seconds until cancelled or jumped to next video
See attached video

**Screenshots (if appropriate)**
Countdown without cancel
https://user-images.githubusercontent.com/1018543/116199247-7d44d600-a769-11eb-9790-c4a394d0d361.mp4

**Testing (for code that is not small enough to be easily understandable)**
Has this pull request been tested?
Please describe shortly how you tested it and whether there are any ramifications remaining. 

**Desktop (please complete the following information):**
 - OS: MacOS
 - OS Version: 10.15.7
 - FreeTube version: 12.0.0 (latest on `development` actually)

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Note: 
The logic in this PR would cause it to NOT display "Playing next video in no time"
Code comment with explanation added

File: src/renderer/views/Watch/Watch.js [hunk 1]
```diff
@@ -876,16 +876,37 @@ export default Vue.extend({
...
+            action: () => {
+              clearTimeout(this.playNextTimeout)
+              clearInterval(countDownIntervalId)
+              this.showToast({
+                message: this.$t('Canceled next video autoplay')
+              })
+            }
+          })
+
+          countDownTimeLeftInMs = countDownTimeLeftInMs - 1000
```
Comment (Svallinn) [inline src/renderer/views/Watch/Watch.js:903]: This variable is completely unnecessary.
You can just use `countDownTimeLeftInSecond` in the conditional logic above.

**Review by Svallinn — CHANGES_REQUESTED** [review]
LGTM. Just make the small change I suggested and I'll merge it.

**Comment by PikachuEXE** [comment]
Requested change made

**Review by Svallinn — APPROVED** [review]

**Pull request merged**